bundle/config/resources/schema.go

```diff
@@ -25,8 +26,15 @@ type Schema struct {
 	URL            string         `json:"url,omitempty" bundle:"internal"`
 }
 
-func (s *Schema) Exists(ctx context.Context, w *databricks.WorkspaceClient, id string) (bool, error) {
-	return false, errors.New("schema.Exists() is not supported")
+func (s *Schema) Exists(ctx context.Context, w *databricks.WorkspaceClient, fullName string) (bool, error) {
+	log.Tracef(ctx, "Checking if schema with fullName=%s exists", fullName)
+
+	_, err := w.Schemas.GetByFullName(ctx, fullName)
+	if err != nil {
+		log.Debugf(ctx, "schema with full name %s does not exist", fullName)
+		return false, err
+	}
+	return true, nil
```

Comment:
What is returned when the schema does not exist, i.e. the 404 response?

This function should discern the schema not existing without an error and the error case.

Reply:
I changed the code to explicitly return nil, nil on 404